frontend/src/api/pipelines/custom.ts

```diff
@@ -135,3 +167,13 @@ export const uploadPipeline: UploadPipelineAPI =
     handlePipelineFailures(
       proxyFILE(hostPath, '/apis/v1beta1/pipelines/upload', fileContents, { name, description }),
     );
+
+export const uploadPipelineVersion: UploadPipelineVersionAPI =
+  (hostPath) => (opts, name, description, fileContents, pipelineId) =>
+    handlePipelineFailures(
+      proxyFILE(hostPath, '/apis/v1beta1/pipelines/upload_version', fileContents, {
+        name,
+        description,
+        pipelineId,
```

Comment:
[Docs](https://www.kubeflow.org/docs/components/pipelines/v1/reference/api/kubeflow-pipeline-api-spec/#operation--apis-v1beta1-pipelines-upload_version-post) say the request param is `pipelineid` - all lowercase.